.gitlab-ci.yml

```diff
@@ -73,6 +74,19 @@ test_subscriptions:
   script:
     - make -j1 tests/test_subscriptions.py
 
+setup_fusillade:
+  stage: fusillade
+  script:
+    - git clone -b master https://github.com/HumanCellAtlas/dcp-fusillade.git
+    - cd dcp-fusillade
+    - source environment
+    - cd ..
+# need to source data-store environment one more time, in case dcp-fusillade overwrites vars
+# (e.g. AWS_DEFAULT_REGION)
+    - source environment
```

Comment:
This is missing:
```
  - if [[ -f "environment.$CI_COMMIT_REF_NAME" ]]; then
  -   source environment.$CI_COMMIT_REF_NAME
  - fi
```

Otherwise, this will always source `dev`.